src/app/activity-data/activity-data.component.html

```diff
@@ -105,7 +105,7 @@
                     </ng-container>
                   </ng-container>
                   <ng-container *ngIf="qDef.selectMode === 'SINGLE'">
-                    <ng-container *ngIf="questionAnswer.answer[0] != null"><b class="TD--Padding-Left-Bigger">{{getUtil().getOptionText(qDef.options,
+                    <ng-container *ngIf="questionAnswer.answer[0] != null && questionAnswer.optionDetails != null"><b class="TD--Padding-Left-Bigger">{{getUtil().getOptionText(qDef.options,
```

Comment:
If `answer` will be empty, we will have `OutOfBoundsException`, or is it checked already somewhere that at this point on 0 index we will have an element?

Reply:
Is checked in L70